src/System.ValueTuple/src/System.ValueTuple.builds

```diff
@@ -6,6 +6,9 @@
     <Project Include="System.ValueTuple.csproj">
       <TargetGroup>portable-net40+sl4+win8+wp8</TargetGroup>
     </Project>
+    <Project Include="System.ValueTuple.csproj">
```

Comment:
thanks @joperezr. what happen when TargetGroup and NugetTargetMoniker are not defined ?